MetaMask/metamask-extension
# feat: Creating a reusable footer component for confirmation pages

## **Description**

The task includes creating a reusable footer component for confirmation pages. This will be very similar to current footer component for confirmation pages.

Designs here: https://www.figma.com/file/uACzbBjwU2tV30UuewYiTx/Confirmation-redesign-V2?type=design&node-id=775-28145&mode=design&t=fcwGIVZS3lkt9rOO-0

The component can be placed in /ui/components/app/confirmation

We need to add logic to cancel / submit the signature request. These functions can be declared inside a hook or the component itself.

## **Related issues**

Fixes: #21278 

## **Manual testing steps**

1. Run yarn storybook, wait for it to load
2. Search for then Footer in App/Confirmation/Footer
3. See the footer component and properties.

## **Screenshots/Recordings**

<img width="999" alt="Screenshot 2023-11-01 at 10 16 49" src="https://github.com/MetaMask/metamask-extension/assets/44811/4532f83c-2b57-4be2-9a1d-8e2a489a7eff">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
